docs/apis/plugintypes/mod_workshop/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Workshop
+tags: []
+---
+:::info
+Work in progress
+:::
+
+import { Since } from '@site/src/components';
+
+<Since versions={["2.0"]} />
+
+This page describes the current implementation of the [Workshop module](https://docs.moodle.org/dev/Workshop_module). See also [Workshop 2.0 specification](https://docs.moodle.org/dev/Workshop_2.0_specification) and [Workshop 2.0 testing](https://docs.moodle.org/dev/Workshop_2.0_testing).
+
+## Overview of Workshop architecture
+
+Rewriting Workshop from scratch for Moodle 2.0 was a great opportunity to use new APIs and approaches to activity module development. In some areas, Workshop was a pioneering component adopting new frameworks - like `File API`, HTML rendering or subplugins support.
```

Comment:
Thsi is just fluff. We don't need to know about 2.0, and the changes from 1.9, etc.

We don't need the Overview title. We just need to know that this page describes the implementation, and that it uses subplugins.
```suggestion
## Architecture
```